agent/xds/clusters.go

```diff
@@ -518,6 +557,37 @@ func (s *Server) makeUpstreamClustersForDiscoveryChain(
 	return out, nil
 }
 
+func injectLBToCluster(l structs.LoadBalancer, c *envoy.Cluster) error {
+	switch l.Policy {
+	case "":
+		return nil
+	case "least_request":
```

Comment:
This is where those `structs` constants would be helpful.

Reply:
Yep, done.